api/app/auto_spatial_advisory/process_hfi.py

```diff
@@ -199,17 +199,17 @@ async def process_hfi(run_type: RunType, run_date: date, run_datetime: datetime,
             # Catch IntegrityError in case these run parameters already exist and continue processing.
             logger.error(e)
 
-        async with get_async_read_session_scope() as session:
-            run_parameters_id = await get_run_parameters_id(session, run_type.value, run_date, for_date)
+        # async with get_async_read_session_scope() as session:
+        #     run_parameters_id = await get_run_parameters_id(session, run_type.value, run_date, for_date)
 
-        async with get_async_read_session_scope() as session:
-            logger.info('Getting high HFI area per zone...')
-            high_hfi_areas = await calculate_high_hfi_areas(session, run_parameters_id)
+        # async with get_async_read_session_scope() as session:
+        #     logger.info('Getting high HFI area per zone...')
+        #     high_hfi_areas = await calculate_high_hfi_areas(session, run_parameters_id)
 
-        async with get_async_write_session_scope() as session:
-            logger.info('Writing high HFI areas...')
-            for row in high_hfi_areas:
-                await write_high_hfi_area(session, row, run_parameters_id)
+        # async with get_async_write_session_scope() as session:
+        #     logger.info('Writing high HFI areas...')
+        #     for row in high_hfi_areas:
+        #         await write_high_hfi_area(session, row, run_parameters_id)
```

Comment:
why is this being removed? My understanding of the problem is that things are slow when retrieving GIS data from the tileserv. This is writing data to our wps db whenever the nats pod receives a new file.

Reply:
This is what I was trying to capture here: https://app.zenhub.com/workspaces/bcws---agile-psu-5e321393e038fba5bbe203b8/issues/gh/bcgov/wps/2512

If we run this code the consumers will block and never finish when trying to write high HFI areas.

Reply:
Since it's not part of MVP / not used, we punted it to investigate later.